### PR TITLE
Added user friendly names to several fields in provenance. Fixes #989

### LINKED ISSUE #989: Add user_friendly name to provenance wherever it is used
Other modules have a user_friendly name where they are used but provenance does not. For example see cell_line.json

No user_friendly:
```
"provenance" : {
            "description": "Provenance information provided by the system.",
            "type": "object",
            "$ref": "system/provenance.json"
        },
```
has user friendly

```
biomaterial_core" : {
            "description": "Core biomaterial-level information.",
            "type": "object",
            "$ref": "core/biomaterial/biomaterial_core.json",
            "user_friendly": "Biomaterial core"
        },
```

When displaying the field in the types the data portal is looking for a user_friendly name and when it cant find one defaults to the field name which looks funny as its lower case.  Would be nice if we can just always have a user friendly for all fields in case they are displayed at some point.

### PR DESCRIPTION
**Pull request by ESapenaVentura**
### Release notes

For `system/provenance` schema:
- Added user friendly name to `submission_date`
- Added user friendly name to `update_date`
- Added user friendly name to `accession`
- Added user friendly name to `submitter_id`
- Added user friendly name to `document_id`
- Added user friendly name to `updater_id`
 

### Reviews requested

- Need 2 Reviewers to approve because this is a minor update
- Last Reviewer to review/approve, merge changes by 11/06/2019

### CODE REVIEW

File: json_schema/system/provenance.json [hunk 1]
```diff
@@ -22,32 +22,38 @@
...
             "type": "string",
-            "format": "date-time"
+            "format": "date-time",
+            "user_friendly": "Submission date"
         },
         "submitter_id": {
             "description": "ID of individual who first submitted project.",
-            "type": "string"
+            "type": "string",
+            "user_friendly": "Submitter id"
```
Comment (malloryfreeberg) [inline json_schema/system/provenance.json]: /id/ID/

File: json_schema/update_log.csv [hunk 2]
```diff
@@ -1 +1,2 @@
 Schema,Change type,Change message,Version,Date
+system/provenance,minor,"Added user friendly names to fields submission_date, update_date, accession, submitter_id, document_id, updater_id. Fixes #989",,
```
Comment (malloryfreeberg) [inline json_schema/update_log.csv]: This is a patch update :)

Comment (ESapenaVentura) [reply json_schema/update_log.csv]: Oh, ok! changing it :)

File: json_schema/system/provenance.json (hunk 1 above)
Comment (ESapenaVentura) [reply json_schema/system/provenance.json]: Done!

**Comment by malloryfreeberg** [comment]
@zperova @ESapenaVentura I'll merge these changes since patch updates only require 1 reviewer.

**Pull request merged**